src/CKAN.jl

```diff
@@ -0,0 +1,60 @@
+struct CKAN <: DataRepo
+end
+
+base_url(::CKAN) = "https://demo.ckan.org/api/3/action/package_show?id="
+
+function description(repo::CKAN, mainpage)
+    desc = mainpage["notes"]
+    authors = [mainpage["author"], mainpage["maintainer"] * " (Maintainer)"]
+    author = format_authors(authors)
```

Comment:
Author and maintainer should be different fields.
The maintainer potentially was not involved with the original data creation. 
Best to just list both seperately